installing/acs-installation-platforms.adoc

```diff
@@ -1,14 +1,81 @@
 :_content-type: ASSEMBLY
 [id="acs-installation-platforms"]
-= Installation platforms and methods
+= Supported platforms and installation methods for RHACS
 include::modules/common-attributes.adoc[]
 :context: acs-installation-platforms
-
 toc::[]
 
 [role="_abstract"]
-{product-title} and {product-title-managed} are supported on various platforms, such as Red Hat OpenShift and Kubernetes platforms. 
+{product-title} ({product-title-short}) and {product-title-managed} ({product-title-managed-short}) are supported on various platforms, such as {osp} and Kubernetes platforms. 
+
+[id="install-platforms-methods_{context}"]
+== Installation methods for different platforms
+
+[role="_abstract"]
+You can perform different types of installations on different platforms.
+
+[NOTE]
+====
+Not all installation methods are supported for all platforms.
+====
+
+.Supported platforms and recommended installation methods
+[%autowidth]
+|===
+|*Platform type footnote:platform[The availability of support for each platform depends on the overarching lifecycle of the platform and the end-of-life date.]*|*Platform*|*Supported for Central*|*Supported for Secured Clusters*|*Supported installation methods*|*Installation steps*
+
+.6+|Managed service platform
+|{osp} Dedicated (OSD)
+|Yes footnote:yesorno[{product-title} ({product-title-short}) is supported as a self-managed add-on for managed OpenShift services that meet the installation and sizing requirements for {product-title-short}. Deploying and managing {product-title-short} components does not fall under the service description for the managed service, and therefore {product-title-short} is not included in the SRE service.]
+|Yes footnote:yesorno[]
+.3+d|Operator (recommended), Helm charts, or roxctl CLI footnote:install[Do not use the `roxctl` installation method unless you have a specific installation that requires this method.]
+.3+a| * xref:../installing/installing_ocp/install-rhacs-ocp.adoc#install-rhacs-ocp[Installing {product-title-short} on {osp}]
+* xref:../installing/installing_cloud_ocp/install-rhacs-cloud-ocp.adoc#install-rhacs-cloud-ocp[Installing ACSCS on {osp}]
```

Comment:
**think:** Are we using ACSCS or RHACS Cloud Service ? What is the title where the link points to? 

**issue:** The link title should match what we have on the linked page. For example this one should be `Setting up RHACS Cloud Service on Red Hat OpenShift`